src/components/card/card.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
```

Comment:
could we uppercase the filename?

Reply:
You mean uppercasing the 'React', right? --> import React from 'React'

If that's the case, I will need to tweak some stuff within the node_modules folder. Do you want me to proceed?

Reply:
No, just the filename i.e. `card.tsx` to `Card.tsx`